src/bootstrap/src/lib.rs

```diff
@@ -77,7 +77,7 @@ const LLD_FILE_NAMES: &[&str] = &["ld.lld", "ld64.lld", "lld-link", "wasm-ld"];
 ///
 /// If you make any major changes (such as adding new values or changing default values), please
 /// ensure that the associated PR ID is added to the end of this list.
-pub const CONFIG_CHANGE_HISTORY: &[usize] = &[115898, 116998, 117435];
+pub const CONFIG_CHANGE_HISTORY: &[usize] = &[115898, 116998, 117435, 116881];
```

Comment:
I'm getting a WARNING pointing me to this PR. Do I need to do something? Or can I keep my existing config? It's unclear to me. The PR description says there's a new config, but why does everyone have to be alerted of its existence?

Reply:
Also this list is unordered now, that seems wrong?

Reply:
update `change-id = 116881` instead may be help. 
AFAIK This is ordered by merge time to keep the source code synchronized with the config file.

Reply:
> update change-id = 116881 instead may be help.

That will silence the warning, sure. But the reason the warning is shown must be that I should do something before silencing it. There's no point in having people mindlessly increment a number every week just to get rid of the latest warning.

Reply:
> Also this list is unordered now, that seems wrong?

https://github.com/rust-lang/rust/blob/1db4b12494f698754b925f55061eb9e6b3241423/src/bootstrap/src/lib.rs#L78-L80

> I'm getting a WARNING pointing me to this PR. Do I need to do something? Or can I keep my existing config? It's unclear to me

https://github.com/rust-lang/rust/blob/1db4b12494f698754b925f55061eb9e6b3241423/src/bootstrap/defaults/config.codegen.toml#L12-L13

PR updated the default value here. I am pretty sure we have people who wants to know this change.

Reply:
Then the PR description needs to say: "if you are coming here from x.py, here's what do do: if you are using the 'codegen' profile, be aware that the default value changed. otherwise, you can ignore this".

Currently I'm coming here clueless and wasting 5min trying to figure out if I should do something, before giving up, frustrated.

Reply:
That's my bad, I will add the description for this PR later